Formula/httpstat.rb

```diff
@@ -0,0 +1,17 @@
+class Httpstat < Formula
+  desc "curl statistics made simple"
+  homepage "https://github.com/reorx/httpstat"
+  url "https://github.com/reorx/httpstat/archive/v1.1.2.tar.gz"
+  sha256 "0468abebfbae9ff7ea35d577b60ff9af9db3ff67ffd81b11ac005c4e9602919b"
+
+  bottle :unneeded
+
+  def install
+    bin.install "httpstat.py"
+    mv bin/"httpstat.py", bin/"httpstat"
+  end
+
+  test do
+    system "#{bin}/httpstat", "https://github.com"
```

Comment:
`brew test` is mostly used by our CI to ensure updates don't break things. When GitHub 500's we probably won't be discussing this anyway 😅 Having a minimum of validation helps us sleep better at night, and by adopting it I won't bother you with "test improvement" again later if I happen to forget what we agreed on :wink: